tests/Utilities/testframework.f90

```diff
@@ -0,0 +1,143 @@
+!> \file
```

Comment:
You probably were not aware but this is not following the code style guide i.e.,

http://cmiss.bioeng.auckland.ac.nz/OpenCMISS/doc/programmer/code_style.html

e.g., CaMeL Case etc.